kahuna/public/js/edits/image-editor.html

```diff
@@ -69,10 +69,8 @@
                 </span>
             </div>
 
-            <ui-archiver
-                class="result-editor__archiver"
-                image="ctrl.image"
-            ></ui-archiver>
+            <gr-image-persist-status class="result-editor__archiver"
+                                     image="ctrl.image"></gr-image-persist-status>
```

Comment:
> `{ image: '=grImage' }`

:+1: 

(will change any other existing custom attributes to this standard as I see them).